gym/monitoring/__init__.py

```diff
@@ -1,3 +1,3 @@
-from gym.monitoring.monitor import Monitor, load_results, monitors as _monitors
+from gym.monitoring.monitor import Monitor, load_results, get_running_monitors
```

Comment:
The intention with `_monitors` is to make clear that this is for use within OpenAI Gym only, and not within user code. I would keep a `_...` name here.

Maybe better to do `get_running_monitors` -> `open_monitors`?